xmtp_mls/src/groups/group_metadata.rs

```diff
@@ -46,6 +46,7 @@ impl GroupMetadata {
             conversation_type: conversation_type as i32,
             creator_inbox_id: self.creator_inbox_id.clone(),
             creator_account_address: "".to_string(), // TODO: remove from proto
+            dm_members: None,
```

Comment:
@neekolas is this right?

Reply:
@cameronvoell is probably the best to ask

Reply:
Hey @rygine that looks correct. 

For context the dm_members libxmtp update [PR](https://github.com/xmtp/libxmtp/pull/901/files) was decided to go on a feature branch, but I forgot to leave a note for future proto updates that we could just set that to None in the mean time. sorry about the confusion.